languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -101590,6 +101590,30 @@ The accident victim died from her injuries.
             <example correction="">This <marker>is</marker> right.</example>
             <example>This <marker>is</marker> conforming with my current knowledge.</example>
         </rule>
+        <rule id="ITS_TO_IT_S_2" name="its -> it's [2]">
+            <antipattern>
+                <token postag="V.*" postag_regexp="yes" />
+                <token>its</token>
+                <token regexp="yes">best|own</token>
+            </antipattern>
+            <antipattern>
+                <token regexp="yes">at|by|on</token>
+                <token>its</token>
+            </antipattern>
+            <pattern raw_pos="yes">
+                <marker>
+                    <token> its</token>
+                </marker>
+                <or>
+                    <token postag="NN|NNS|NN:U|NN:UN|SENT_END|PARA_END" postag_regexp="yes" negate_pos="yes">
+                        <exception>hindlimbs</exception>
+                    </token>
+                    <token regexp="yes">anything|everything|something|nothing|anybody|everybody|somebody|nobody</token>
```

Comment:
I've quickly tested this in the rule editor by using first one, then the other version of the token (thereby avoiding using `<or>`).

I really like the rule with the second token `<token regexp="yes">anything|everything|something|nothing|anybody|everybody|somebody|nobody</token>`. It's unbelievable we don't find those sentences yet!

With the first token (`<token postag="NN|NNS|NN:U|NN:UN|SENT_END|PARA_END" postag_regexp="yes" negate_pos="yes"><exception>hindlimbs</exception></token>`), there are many false positives, though:

- TransAlta Corp. sold its retail business last year

Always make sure that you test thoroughly. For each version of your pattern, put at least one example. Look for false positives and write antipatterns if needed. For each antipattern, put in an example to see whether it actually works.